src/debugger/debugAdapter.ts

```diff
@@ -24,6 +29,12 @@ export class DebugAdapter implements DebugAdapterTracker {
                     break;
                 case DEBUG_COMMANDS.STACK_TRACE:
                     this.messagingService.sendPauseMessage();
+                    break;
+                case DEBUG_COMMANDS.DISCONNECT:
+                    // Triggered on stop event for debugger
+                    if (!message.arguments.restart) {
+                        this.debugCommunicationService.handleStopEvent();
+                    }
```

Comment:
I would put a break statement at the end of the case, so the next developer doesn't have to remember to check if the previous case has a break or not.